.github/workflows/license_check.yml

```diff
@@ -0,0 +1,18 @@
+name: Check license of dependencies
+on: [push, pull_request]
```

Comment:
Just a thought here. Do we need the workflow to run on push? It seems to me like any new module additions or updates would have to go though PRs either way.

Reply:
I feel it is a nice safety measure just in case, "belt and suspender" kind of approach.